tests/test_rendering.py

```diff
@@ -46,7 +46,9 @@ def test_example_message_round_trippable(data):
     example = imp.module_from_spec(spec)
     sys.modules['example'] = example
     spec.loader.exec_module(example)
-    original_example_avro_model = data.draw(st.from_type(example.ExampleAvroModel))
+    st.register_type_strategy(Decimal, st.decimals(allow_nan=False, allow_infinity=False, places=2))
```

Comment:
I see a couple issues here:
1. Per https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.register_type_strategy , this appears to register the type strategy globally, rather than just for this test (or one run thereof). This presumably means that other tests attempting to draw `Decimal` are affected, which seems like behavior we want to avoid; alternatively, if we _did_ want this to be overridden globally for the entire test suite, it seems like we'd want it in a `conftest.py` or similar as opposed to buried in a single test.
2. If we limit the space for `Decimal`, I think it's only fair that we remove the now-redundant `assume` calls from https://github.com/hchasestevens/avro2py/pull/15/files#diff-530cd1c39b1adde7f11281b896978de50dae64009a7589c4f3c5ede8c76f1005R55-R60 

Rather than this, can we not modify https://github.com/hchasestevens/avro2py/pull/15/files#diff-530cd1c39b1adde7f11281b896978de50dae64009a7589c4f3c5ede8c76f1005R62-R63 to just set the `decimal` and `maybeDecimal` attributes to draws of `st.decimals(allow_nan=False, allow_infinity=False, places=2)` explicitly?

Reply:
I didn't realize `st` was global, but looking at the fact that it's just a renamed import now that makes sense.

So sure in that case I think it makes sense to not do it that way.